api_tests/registrations/views/test_registration_linked_nodes.py

```diff
@@ -1,395 +1,457 @@
         nodes_returned = [linked_node['id'] for linked_node in res.json['data']]
-        assert_equal(len(nodes_returned), len(self.node_ids))
+        assert len(nodes_returned) == len(node_ids)
 
-        for node_id in self.node_ids:
-            assert_in(node_id, nodes_returned)
+        for node_id in node_ids:
+            assert node_id in nodes_returned
 
         # Disconnect contributor_removed so that we don't check in files
         # We can remove this when StoredFileNode is implemented in osf-models
         with disconnected_from_listeners(contributor_removed):
-            self.linked_node2.remove_contributor(user, auth=self.auth)
-            self.public_node.remove_contributor(user, auth=self.auth)
+            private_node_two.remove_contributor(user, auth=auth)
+            public_node.remove_contributor(user, auth=auth)
 
-        res = self.app.get(
+        res = app.get(
             '/{}registrations/{}/linked_nodes/'.format(API_BASE, new_linking_registration._id),
             auth=user.auth
         )
         nodes_returned = [linked_node['id'] for linked_node in res.json['data']]
-        assert_equal(len(nodes_returned), len(self.node_ids) - 1)
+        assert len(nodes_returned) == len(node_ids) - 1
 
-        assert_in(self.linked_node._id, nodes_returned)
-        assert_in(self.public_node._id, nodes_returned)
-        assert_not_in(self.linked_node2._id, nodes_returned)
+        assert private_node_one._id in nodes_returned
+        assert public_node._id in nodes_returned
+        assert private_node_two._id not in nodes_returned
 
-    def test_linked_nodes_doesnt_return_deleted_nodes(self):
-        self.linked_node.is_deleted = True
-        self.linked_node.save()
-        res = self.app.get(self.url, auth=self.user.auth)
+    def test_linked_nodes_doesnt_return_deleted_nodes(self, app, user, url, private_node_one, private_node_two, public_node, node_ids):
+        private_node_one.is_deleted = True
+        private_node_one.save()
+        res = app.get(url, auth=user.auth)
 
-        assert_equal(res.status_code, 200)
+        assert res.status_code == 200
         nodes_returned = [linked_node['id'] for linked_node in res.json['data']]
-        assert_equal(len(nodes_returned), len(self.node_ids) - 1)
+        assert len(nodes_returned) == len(node_ids) - 1
 
-        assert_not_in(self.linked_node._id, nodes_returned)
-        assert_in(self.linked_node2._id, nodes_returned)
-        assert_in(self.public_node._id, nodes_returned)
+        assert private_node_one._id not in nodes_returned
+        assert private_node_two._id in nodes_returned
+        assert public_node._id in nodes_returned
 
-    def test_attempt_to_return_linked_nodes_logged_out(self):
-        res = self.app.get(
-            self.url, auth=None,
-            expect_errors=True
-        )
+    def test_attempt_to_return_linked_nodes_logged_out(self, app, url):
+        res = app.get(url, auth=None, expect_errors=True)
```

Comment:
I mostly was looking for consistency since there are other places in this file that don't put it explicitly.